### PR TITLE
test: Add a couple of project setup e2e tests

### PR DESCRIPTION
**Pull request by BlueWinds**
- Apart of #19051

### User facing changelog
None, just additional tests

### Additional details
Covers a couple of project setup cases from the [feature checklist](https://docs.google.com/spreadsheets/d/1iPwi89aW6aYeA0VT1XOhYdAWLuScW0okrlfcL9fzh3s/edit#gid=0).

### PR Tasks
- [x] Have tests been added/updated?
- [N/A] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [N/A] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [N/A] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [N/A] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/launchpad/cypress/e2e/integration/project-setup.spec.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    cy.get('h1').should('contain', WIZARD_STEPS.find((s) => s.type === 'welcome').title)
+    cy.get('h2').should('contain', 'E2E Testing')
+    cy.get('h2').should('contain', 'Component Testing')
+  })
+
+  it('welcome page has button to learn about testing types', () => {
+    cy.setupE2E('todos')
+    cy.visitLaunchpad()
+
+    cy.get('button.hocus-link-default')
```
Comment (marktnoonan) [inline packages/launchpad/cypress/e2e/integration/project-setup.spec.ts:18]: I'd prefer to not depend on a specific css class when finding the button. We could just remove it, or possibly shorten it all to:

```
cy.contains('button', defaultMessages.welcomePage.review).click()
```

**Review by marktnoonan — CHANGES_REQUESTED** [review]
There's one small change I'm requesting - to remove `.hocus-link-deafult`. 

Also it might be good to add `be.visible` assertions when checking elements that the test don't do any other interaction with. I don't think we do that consistently at the moment, but it helps confirm the element is seen on the page and not just present in the DOM.

File: packages/launchpad/cypress/e2e/integration/project-setup.spec.ts (hunk 1 above)
Comment (BlueWinds) [reply packages/launchpad/cypress/e2e/integration/project-setup.spec.ts:18]: Today I learned about `cy.contains()`! Will change to use that instead.

**Review by tbiethman — APPROVED** [review]
Assertions look good; I think we just need to rebase this on top of the recent changes.

**Review by marktnoonan — DISMISSED** [review]

**Review by tbiethman — APPROVED** [review]

**Comment by BlueWinds** [comment]
Lots of tests are failing, but they seem to also be failing on the 10.0-release branch, unrelated to the changes here. Both files this PR touches pass.

**Review by tbiethman — APPROVED** [review]

**Review by davidmunechika — APPROVED** [review]
lgtm!

**Pull request merged**